### DISCUSSION TITLE
使用@quick-start/electron，然后pnpm i 安装包一直开着很久然后报错

### DISCUSSION THREAD

**Discussion by jiahaowu-git** [alex8088/electron-vite#292] [Q&A]
![Snipaste_2023-09-25_15-26-27](https://github.com/alex8088/electron-vite/assets/8285267/4c066165-ed4b-442f-8671-c6678f2ba1dd)
node版本：v20.5.1
pnpm版本：8.7.1

**Comment by alex8088** [comment]
网络问题，脚手架命令最后选项开启electron镜像下载

**Comment by jiahaowu-git** [comment]
> 网络问题，脚手架命令最后选项开启electron镜像下载
好的
